pkg/helm/chartproxy/repos_test.go

```diff
@@ -341,3 +341,148 @@ func TestHelmRepo_IndexFile(t *testing.T) {
 		})
 	}
 }
+
+
+func TestHelmRepoGetter_SkipDisabled(t *testing.T) {
+	tests := []struct {
+		name             string
+		helmCRS          []*unstructured.Unstructured
+		expectedRepoName []string
+		disabled	bool
+		apiErrors        []apiError
+	}{
+		{
+			name: "list helm repos with no disabled repo should list all repositories",
```

Comment:
This test is not needed because it is already covered `TestHelmRepoGetter_List` function.

Reply:
Makes sense, removed